app/src/mainnet_staging/res/values/static_strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name" translatable="false">Safe Multisig - Mainnet Staging</string>
```

Comment:
Do you see the end of this name in your home screen?

Maybe we could rename it to Mainnet Staging - Safe Multisig?

Reply:
or just Mainnet Staging. It is clear anyway that the app is Safe Multisig

Reply:
but this name is mainly for app distribution screen. App is identifiable by the icon and nobody should use it, but us. So I think it's fine to just leave it as is and continue using our established app naming schema :)

Reply:
So you vote for Mainnet Staging?